src/deprecated.js

```diff
@@ -22,9 +23,39 @@ jQuery.fn.extend( {
 		return arguments.length === 1 ?
 			this.off( selector, "**" ) :
 			this.off( types, selector || "**", fn );
+	},
+
+	hover: function( fnOver, fnOut ) {
+		return this.mouseenter( fnOver ).mouseleave( fnOut || fnOver );
 	}
 } );
 
+jQuery.each( ( "blur focus focusin focusout resize scroll click dblclick " +
+	"mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave " +
+	"change select submit keydown keypress keyup contextmenu" ).split( " " ),
+	function( _i, name ) {
+
+		// Handle event binding
+		jQuery.fn[ name ] = function( data, fn ) {
+			return arguments.length > 0 ?
+				this.on( name, null, data, fn ) :
+				this.trigger( name );
+		};
+	} );
+
+jQuery.each( [
```

Comment:
I think I need to wrap it with an `if ( jQuery.ajax )` condition to better support custom builds.

Reply:
Correction: I'll instead create a `src/deprecated` directory and will move it there. This, together with a proper modification of the [`removeWith` section in Gruntfile.js](https://github.com/jquery/jquery/blob/c1ee33aded44051b8f1288b59d2efdc68d0413cc/Gruntfile.js#L60-L69) will allow to cut this code out when excluding AJAX.

The same has to be done for the event alias part.

Reply:
Done.